gradle/libs.versions.toml

```diff
@@ -4,6 +4,7 @@ androidGradlePlugin = "8.2.2"
 androidx-activity-compose = "1.9.0-alpha03"
 androidx-appcompat = "1.6.1"
 androidx-compose-bom = "2024.02.01"
+androidx-compose-ui-test = "1.7.0-alpha03"
```

Comment:
how about adding a maintenance comment here to remove this version when the bom contains 1.7.0?